management/management_request.go

```diff
@@ -285,3 +285,13 @@ func Stringify(v interface{}) string {
 	}
 	return string(b)
 }
+
+// Sort configures a request to sort data by the selected field.
+// Use 1 to sort ascending and -1 to sort descending.
```

Comment:
Should we use a boolean for the public API, and translate to "1" or "-1" internally? Because it's a two-state parameter.